cmd/minishift/cmd/root.go

```diff
@@ -283,12 +283,17 @@ func initializeProfile() string {
 	return ""
 }
 
+func setViperDefaultKeyValues() {
```

Comment:
@budhrg I would suggest renaming the function as `setViperConfigDefaults` as `KeyValues` in `setViperDefaultKeyValues` does not make much sense to me.